pkg/scheduler/plugins/multihierarchyelasticquota/quota_info.go

```diff
@@ -0,0 +1,184 @@
+/*
+Copyright 2022 The Koordinator Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package multihierarchyelasticquota
+
+import (
+	"sync"
+
+	"github.com/koordinator-sh/koordinator/apis/scheduling/v1alpha1"
+	"github.com/koordinator-sh/koordinator/pkg/util"
+	v1 "k8s.io/api/core/v1"
+	v12 "k8s.io/apiserver/pkg/quota/v1"
+)
+
+type QuotaCalculateInfo struct {
+	Max    v1.ResourceList `json:"max"`
+	OriMin v1.ResourceList `json:"ori_min"`
+	//在quotaTree中实际生效的minQuota
```

Comment:
notes should in english; and CalMin -> AutoScaledMin

Reply:
done

Reply:
please resolve conversation after fix the comments